participant-manager-datastore/participant-manager-service/src/main/resources/application.properties

```diff
@@ -82,7 +80,7 @@ management.endpoints.enabled-by-default=true
 management.endpoints.web.exposure.include=logfile,loggers
 
 update.user.subject=Your account has been updated
-update.user.body=<html><body>Hi {{FIRST_NAME}},<BR><BR>This is to inform you that one or more details of your user account have been updated by the administrator in the {{ORG_NAME}} MyStudies Participant Manager portal.<BR><BR>You may be required to sign in to the portal again in case you were in an active session.<BR><BR>For any questions or assistance, please write to {{CONTACT_EMAIL_ADDRESS}} <BR><BR>Thanks,<BR>The {{ORG_NAME}} MyStudies support team<BR>---------------------------------------------------------------------------------------------------------------------------<BR>Note: This is an auto-generated email. Please do not reply.</body></html>
+update.user.body=<html><body>Hi {{FIRST_NAME}},<BR><BR>This is to inform you that one or more details of your admin account have been updated by the superadmin user in the {{ORG_NAME}} MyStudies Participant Manager portal.<BR><BR>You may be required to sign in to the portal again in case you were in an active session.<BR><BR>For any questions or assistance, please write to {{CONTACT_EMAIL_ADDRESS}} <BR><BR>Thanks,<BR>The {{ORG_NAME}} MyStudies support team<BR>---------------------------------------------------------------------------------------------------------------------------<BR>Note: This is an auto-generated email. Please do not reply.</body></html>
```

Comment:
We can probably remove the text "by the superadmin user"

Reply:
Fixed Review comment.